src/extlibs/er_force_sim/src/amun/simulator/simulator.cpp

```diff
@@ -318,8 +317,31 @@ void Simulator::handleSimulatorTick(double time_s)
                 &ErrorAggregator::aggregate);
     }
 
+    // find out if ball and any robot collide
+    bool ball_collision = false;
+    for (const auto &pair : m_data->robotsBlue)
```

Comment:
seems like you could use something like https://cplusplus.com/reference/algorithm/any_of/

Reply:
updated